.github/code-of-conduct.md

```diff
@@ -0,0 +1,78 @@
+
+Though we welcome people fluent in all languages, Cython development is conducted in English.
+
+Standards for behaviour in the Cython community are detailed in the Code of Conduct above. Participants in our community should uphold these standards in all their interactions and help others to do so as well (see next section).
+
+## Reporting Guidelines
+
+We know that it is painfully common for internet communication to start at or devolve into obvious and flagrant abuse. We also recognize that sometimes people may have a bad day, or be unaware of some of the guidelines in this Code of Conduct. Please keep this in mind when deciding on how to respond to a breach of this Code.
+
+For clearly intentional breaches, report those to the Code of Conduct Committee (see below). For possibly unintentional breaches, you may reply to the person and point out this code of conduct (either in public or in private, whatever is most appropriate). If you would prefer not to do that, please feel free to report to the Code of Conduct Committee directly, or ask the Committee for advice, in confidence.
+
+You can report issues to the Cython Code of Conduct Committee at cython-conduct@googlegroups.com.
+
+Currently, the Committee consists of:
+
+* Stefan Behnel
+* Robert Bradshaw
+* Ralf Gommers
+
+If your report involves any members of the Committee, or if they feel they have a conflict of interest in handling it, then they will recuse themselves from considering your report. Alternatively, if for any reason you feel uncomfortable making a report to the Committee, then you can also contact senior NumFOCUS staff at [conduct@numfocus.org](https://numfocus.org/code-of-conduct#persons-responsible).
+
+## Incident reporting resolution & Code of Conduct enforcement
+
+_This section summarizes the most important points, more details can be found in_ [Cython Code of Conduct - How to follow up on a report](report-handling-manual.md).
+
+We will investigate and respond to all complaints. The Cython Code of Conduct Committee will protect the identity of the reporter, and treat the content of complaints as confidential (unless the reporter agrees otherwise).
+
+In case of severe and obvious breaches, e.g. personal threat or violent, sexist or racist language, we will immediately disconnect the originator from Cython communication channels; please see the manual for details.
+
+In cases not involving clear severe and obvious breaches of this Code of Conduct the process for acting on any received Code of Conduct violation report will be:
+
+1. acknowledge report is received,
```

Comment:
I guess we should have a couple of standard email phrases prepared for that, including some words about this procedure, so that people who actually find the guts to report something don't drop into an unprepared ad-hoc communication channel.

Reply:
Great point. That should not be inside this manual I guess (would be weird to see for a reporter), but it will be very handy to have stored somewhere (maybe in an email to the cython-conduct list). I'll see if there's already such a list of phrases from another project that we can use as a starting point.